smithy-jsonschema/src/test/java/software/amazon/smithy/jsonschema/JsonSchemaConverterTest.java

```diff
@@ -780,4 +781,55 @@ public void intEnumsCanBeDisabled() {
                 IoUtils.toUtf8String(getClass().getResourceAsStream("int-enums-disabled.jsonschema.v07.json")));
         Node.assertEquals(document.toNode(), expected);
     }
+
+    @Test
```

Comment:
Can you please add explicit test cases for both of these not being applied on Draft 7?

Reply:
[9c7fde9](https://github.com/smithy-lang/smithy/pull/2221/commits/9c7fde97fb8150d195ad126727b6c5d6af4c2565)